RELEASING.md

```diff
@@ -43,7 +43,6 @@ For Squares, membership is managed through the `Workflow Swift Owners` registry
     bundle exec pod trunk push WorkflowRxSwift.podspec --synchronous
     bundle exec pod trunk push WorkflowReactiveSwiftTesting.podspec --synchronous
     bundle exec pod trunk push WorkflowRxSwiftTesting.podspec --synchronous
-    bundle exec pod trunk push WorkflowSwiftUI.podspec --synchronous
```

Comment:
Do we maybe want to update this to explain how you release a new version of WorkflowSwiftUI?
We are currently blocked from doing a Workflow release with Xcode 15 because of an issue in cocoapods.  It would probably benefit us to move all of workflow to package manager if we have to move part of it.

Reply:
No I think doing that separately would be good.